### PR TITLE
Feature/dropdown ax

### PR DESCRIPTION
**Pull request by scriby**
#### Pull request checklist

- [x] Addresses an existing issue: #1603
- [x] Include a change request file if publishing 

#### Description of changes

Fixes an issue with screenreaders not reading menu items from the dropdown component. Uses the structure from https://www.w3.org/TR/wai-aria/roles#combobox as an example.

I tested manually in JAWS + IE11, JAWS + Chrome, and VoiceOver + Chrome on OSX.

### CODE REVIEW

**Comment by scriby** [comment]
I've already signed the cla, so not sure why that label was added.

File: packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -311,9 +313,9 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           }
         ) }
         onClick={ () => this._onItemClick(item.index) }
-        role='menu'
+        role='option'
         aria-selected={ this.state.selectedIndex === item.index ? 'true' : 'false' }
-        aria-label={ item.text }
+        ariaLabel={ item.text }
```
Comment (scriby) [inline packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx]: I created https://github.com/OfficeDev/office-ui-fabric-react/issues/1621 for this "issue"

File: packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -253,6 +254,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
         className={ css('ms-Dropdown-items', styles.items) }
         aria-labelledby={ id + '-label' }
         onKeyDown={ this._onZoneKeyDown }
+        role='listbox'
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx]: not sure about this, is a combobox a listbox?

Comment (scriby) [inline packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx]: No.

The combobox is the main control that indicates what is selected, and can pop open a list of choices.

The list of choices that the combobox shows is a listbox (with options).

This is the same structure used in the example I linked.

Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx]: Oh there was a link.. :) Makes sense then, thanks for the change!

**Comment by scriby** [comment]
I think this is ready to merge!

**Comment by cschleiden** [comment]
@scriby Was waiting for the build last night, but it took forever. I'll try again :)

**Pull request merged**